client/src/components/Table/Table.jsx

```diff
@@ -90,7 +90,7 @@ class Table extends Component {
                         <>
                           <input
                             type="checkbox"
-                            checked={isChecked}
+                            checked={this.state.exportSome}
```

Comment:
There is no such exportSome state in this component so it's always undefined.
You are still giving the isChecked props in TopicData line 1178 but you removed it here